core/sql/executor/ExHbaseAccess.cpp

```diff
@@ -3033,14 +3033,23 @@ void ExHbaseAccessTcb::setRowID(char *rowId, Lng32 rowIdLen)
    memcpy(rowID_.val, rowId, rowIdLen);
 }
 
-void ExHbaseAccessTcb::buildLoggingPath(
-                             const char * loggingLocation,
-                             char * logId,
-                             const char * tableName,
+void ExHbaseAccessTcb::buildLoggingFileName(
+                             const char * currCmdLoggingLocation,
+                             const char *tableName,
                              const char * loggingFileNamePrefix,
                              Lng32 instId,
                              char * loggingFileName)
 {
+  sprintf(loggingFileName, "%s/%s_%s_%d",
```

Comment:
Yes. There is a possibility of buffer overflow because the size of the buffer is not calculated. Will fix it.